src/expression_file_generator.py

```diff
@@ -84,29 +85,28 @@ def generate_file(self):
 
             references = ','.join(ref_obj['pubMedId'] or ref_obj['pubModId'] for ref_obj in expression['References']) or ''
```

Comment:
this `or` expression seems to be redundant (perhaps just want: `ref_obj['pubMedId']` ?)